config.example.wikimedia.yaml

```diff
@@ -148,6 +149,91 @@ templates:
                 request:
                   uri: http://graphoid.wikimedia.org/{domain}/v1/png/{title}/{revision}/{graph_id}
 
+      /{module:summary}:
+        x-subspec:
+          info: Summary sys module API
+          paths:
+            /request/{title}:
+              get:
+                x-setup-handler:
+                  - init:
+                      method: 'put'
+                      uri: /{domain}/sys/key_value/summary
+                      body:
+                        valueType: 'json'
+                        version: 1
+                x-request-handler:
+                  - get_revision:
+                      request:
+                        method: get
+                        uri: /{domain}/sys/page_revisions/page/{title}
+                        headers:
+                          cache-control: '{cache-control}'
+                  - process_request:
+                      request:
+                        method: get
+                        uri: /{domain}/sys/summary/handler/{$$.default($.request.headers.cache-control, 'none-given')}
+                        body: '{$.get_revision.body.items[0]}'
+            /handler/no-cache:
+              get:
+                x-request-handler:
+                  - get_extracts:
+                      request:
+                        method: post
+                        uri: /{domain}/sys/action/query
+                        body:
+                          prop: 'extracts|pageimages'
+                          redirects: 'true'
+                          exsentences: 5
+                          explaintext: 'true'
+                          piprop: 'thumbnail'
+                          pithumbsize: 320
+                          titles: '{$.request.body.title}'
```

Comment:
The `/handler/no-cache` route has no parameter named `title`. This should be taken from the body, also because the title supplied by the user might be a redirect.